frontend/src/views/v-zoom.vue

```diff
@@ -101,8 +101,9 @@
       .hash
         span {{ slice.hash.substr(0, 7) }}
       span.fileTypeLabel(
+        v-if="containsAtLeastOneSelected(Object.keys(slice.fileTypesAndContributionMap))"
```

Comment:
Try adding a comma here to address the failing CI.

Reply:
Thank you, addressed it for my next commit!